### DISCUSSION TITLE
Open split with specified size from script?

### DISCUSSION THREAD

**Discussion by Sonico98** [kovidgoyal/kitty#6166] [Q&A] [answered]
I have a shell script which opens a kitty split to run some command, and would like the split to occupy about 1/3 of the total screen height.
Right now I'm running this inside my script to accomplish what I want:
```
# cat is just an example command
id="$(kitty @ launch --location split cat)"

kitty @ resize-window -m id:"$id" -a vertical -i -100
kitty @ resize-window -m id:"$id" -a vertical -i 5
```
While it works, it makes the split "jump" around the screen very quickly (it first opens occupying half of the screen, then "jumps" to the bottom of the screen, and finally it makes itself a bit taller).

Reading the docs I've seen that layouts can be configured inside kitty.conf to always open with certain sizes, but I only want them to open with a certain size when launching them from the script.

Is there some other way to do this? Thanks

**Comment by kovidgoyal** [comment, marked as answer]
Define the layout with the bias you want as one of the enabled_layouts.
Then use kitty @ goto-layout ...
to switch to it.

Note that you have to run kitty nightly for goto-layout to work with
layouts using options.

**Reply by Sonico98** [reply]
Thanks!
